src/lib/tvgPaint.cpp

```diff
@@ -298,7 +298,7 @@ Paint* Paint::duplicate() const noexcept
 Result Paint::composite(std::unique_ptr<Paint> target, CompositeMethod method) const noexcept
 {
     if (pImpl->composite(target.release(), method)) return Result::Success;
-    return Result::InsufficientCondition;
```

Comment:
which scenario would occur this ? any case?

"A possible error is due to erroneous arguments"

Reply:
e.g. ```composite(nullptr, CompositeMethod::AlphaMask)``` or ```composite(target, CompositeMethod::None)```
so in cases, when the args makes no sense.
in your opinion it should be InsufficientCondition instead InvalidArguments?

Reply:
@mgrudzinska ok, you're correct.